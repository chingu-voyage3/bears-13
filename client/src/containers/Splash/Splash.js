import React from 'react';
import { NavLink } from 'react-router-dom';

import './splash.css';

import { BigText, MedText, BigDiv } from '../../utils/shared-styles';

const Splash = () => (
  <BigDiv>
    <BigText id="mobile-title">Fair Rents</BigText>
    <MedText id="mobile-sub">Rent reviews for and by renters</MedText>
    <NavLink id="big-link" to="/search">
      Search for a property now
    </NavLink>
  </BigDiv>
);
// class Splash extends React.Component {
//   constructor() {
//     super();
//     this.state = {
//       user: null,
//     };
//   }
//   render() {
//     return (
//       <BigDiv>
//         <BigText id="mobile-title">Fair Rents</BigText>
//         <MedText id="mobile-sub">Rent reviews for and by renters</MedText>
//         <NavLink id="big-link" to="/search">
//           Search for a property now
//         </NavLink>
//       </BigDiv>
//     );
//   }
// }

export default Splash;
