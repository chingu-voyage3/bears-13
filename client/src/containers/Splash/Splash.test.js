/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import expect from 'expect';
import { BrowserRouter as Router } from 'react-router-dom';

import Splash from './Splash';

const jsxComponent = (
  <Router>
    <Splash />
  </Router>
);
describe('SplashComponent', () => {
  let tree;
  beforeAll(() => {
    const component = renderer.create(jsxComponent);
    tree = component.toJSON();
  });

  it('should render', () => {
    expect(tree).toMatchSnapshot();
  });

  it('should render "Fair Rents"', () => {
    expect(tree.children[0].children[0]).toEqual('Fair Rents');
  });
});
