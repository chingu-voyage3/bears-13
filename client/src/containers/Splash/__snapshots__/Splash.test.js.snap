// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplashComponent should render 1`] = `
<div
  className="sc-bxivhb leFDWK"
>
  <p
    className="sc-bdVaJa fuxuhi"
    id="mobile-title"
  >
    Fair Rents
  </p>
  <p
    className="sc-bwzfXH gllGzP"
    id="mobile-sub"
  >
    Rent reviews for and by renters
  </p>
  <a
    aria-current={false}
    className={undefined}
    href="/search"
    id="big-link"
    onClick={[Function]}
    style={undefined}
  >
    Search for a property now
  </a>
</div>
`;
